tmt-app/src/Styling/CustomStyling.js

```diff
@@ -45,6 +45,9 @@ export const CustomTheme = createTheme({
     Avacado: {
       main: "#F0F6E2",
     },
+    Text: {
+      main: "#1B1D2E",
+    },
```

Comment:
Left a comment about where this color should be https://mui.com/material-ui/customization/theming/

Take a look at MUI docs and see how our theme extension differs from the doc examples